src/gui/properties/peerlistwidget.cpp

```diff
@@ -140,7 +139,6 @@ PeerListWidget::PeerListWidget(PropertiesWidget *parent)
 PeerListWidget::~PeerListWidget()
 {
     saveSettings();
-    delete m_proxyModel;
```

Comment:
It might be of no consequence in this particular case, but conceptually this is wrong.  The `m_proxyModel` object has to be deleted before `m_listModel` or one has to call `m_proxyModel->setSourceModel(nullptr)` here.